apps/hoarder/config.json

```diff
@@ -0,0 +1,39 @@
+{
+
+    "$schema": "../schema.json",
+    "name": "Hoarder",
+    "id": "hoarder",
+    "available": true,
+    "short_desc": "A self-hostable bookmark-everything app (links, notes and images) with AI-based automatic tagging and full text search",
+    "author": "https://github.com/hoarder-app",
+    "port": 8685,
+    "categories": ["utilities", "ai"],
+    "description": "A self-hostable bookmark-everything app (links, notes and images) with AI-based automatic tagging and full text search",
+    "tipi_version": 6,
+    "version": "v1.6.0",
+    "source": "https://github.com/hoarder-app/hoarder",
+    "form_fields": [
+        {
+            "type": "random",
+            "label": "Nextauth Secret",
+            "min": 20,
+            "env_variable": "NEXTAUTH_SECRET"
+        },
+        {
+            "type": "random",
+            "label": "Meili master key",
+            "min": 20,
+            "env_variable": "MEILI_MASTER_KEY"
+        },
+        {
+            "type": "text",
+            "label": "OpenAI API Key",
+            "required": false,
+            "env_variable": "OPENAI_API_KEY"
+        }
+    ],
+    "exposable": true,
+    "supported_architectures": ["amd64", "arm64"],
+    "created_at": 1726034457000,
+    "updated_at": 1726034457000
```

Comment:
These two should be the actual date that you added the app, you can find the value by right clicking on any page (even this), clicking inspect, going to the console option, typing `new Date().getTime()` and just copy paste the value on both fields.